pages/web-sites.md

```diff
@@ -120,8 +120,14 @@ For other sites:
 
 To create a tag, use `npm version [major | minor | patch]`.
 
-Afterwards, make sure to push both version change commit and the tag:
-`git push --tags origin master`
+Afterwards, make sure to push both version change commit and the tag to the main jQuery repo.
+
+<div class="warning">You'll want to check your remote setup using ```git remote -v``` and check which remote is used for the main jQuery repo.</div>
```

Comment:
@arthurvr It all looks fine for me when deployed locally. Should I use single ``` ` ``` instead?